content/en/docs/reference/config/analysis/ist0157/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: InvalidTelemetryProvider
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+This message occurs when telemetry with empty providers will be ignored if the default providers in MeshConfig is empty.
```

Comment:
```suggestion
This message occurs when a Telemetry resource with empty providers is set, and it will be ignored if the default providers in MeshConfig are empty.
```

Reply:
done